packages/analytics/analytics-metric-provider/src/components/metricsProviderUtil.ts

```diff
@@ -112,7 +112,7 @@ export const defaultFetcherDefs = (opts: FetcherOptions) => {
     tz,
 
     // Don't query latency trends in the multi-entity case: it's possible, but wasteful.
-    withTrend: hasTrendAccess && !multiEntityQuery,
+    withTrend: computed(() => hasTrendAccess.value && !multiEntityQuery),
```

Comment:
nit: can you add a return type to the computed?

Reply:
Will do.

Reply:
Done.